examples/micronaut/build.gradle

```diff
@@ -1,45 +1,55 @@
 plugins {
-    id 'application'
-    id 'groovy'
-    id 'io.spring.dependency-management' version '1.0.6.RELEASE'
-    id 'net.ltgt.apt-idea' version '0.18'
+    id "groovy"
+    id "com.github.johnrengelman.shadow" version "5.2.0"
+    id "application"
```

Comment:
nit: is there a rule for using single quotes and double quotes? Doesn't seem like the usage is consistent across the file.

Reply:
Actually, no rule. Often you use double quotes for interpolated strings, and use single quotes for constant strings. But it's not a hard rule at all. 
I took the recent `build.gradle` generated by Micronaut Launch (the project generator), and it was what it gave me :-)

Reply:
Gotcha! Merging it now. Thanks!